sbt-dumpschema/src/main/scala/org/virtuslab/akkasaferserializer/DumpSchemaPlugin.scala

```diff
@@ -0,0 +1,34 @@
+package org.virtuslab.akkasaferserializer
+
+import sbt.Keys._
+import sbt.{Def, _}
+
+object DumpSchemaPlugin extends AutoPlugin {
+
+  object autoImport extends DumpSchemaKeys {
+    val baseDumpSchemaSettings: Seq[Def.Setting[_]] = DumpSchemaPlugin.baseDumpSchemaSettings
+  }
+  import autoImport.{baseDumpSchemaSettings => _, _}
+
+  override def projectSettings: Seq[Def.Setting[_]] = additionalSettings ++ dumpSchemaSettings
+
+  override def globalSettings: Seq[Def.Setting[_]] = Nil
+
+  lazy val additionalSettings: Seq[Def.Setting[_]] = Seq(
+    libraryDependencies += compilerPlugin((dumpSchema / dumpSchemaPlugin).value),
+    scalacOptions += s"-P:dump-schema-plugin:--file ${(dumpSchema / dumpSchemaPluginOutput).value.toPath}",
+    scalacOptions += (if ((dumpSchema / dumpSchemaPluginVerbose).value) "-P:dump-schema-plugin:-v" else ""))
+
+  lazy val dumpSchemaSettings: Seq[Def.Setting[_]] = baseDumpSchemaSettings
+
+  lazy val baseDumpSchemaSettings: Seq[Def.Setting[_]] = Seq(
+    DumpSchema.assemblyTask(dumpSchema),
+    dumpSchema / dumpSchemaOutputPath := { target.value / (dumpSchema / dumpSchemaFilename).value },
+    dumpSchema / dumpSchemaFilename := (dumpSchema / dumpSchemaFilename)
+        .or(Def.setting(s"${name.value}-dumpschema-${version.value}.json"))
+        .value,
+    dumpSchema / dumpSchemaPlugin := "org.virtuslab" %% "sbt-dumpschema-plugin" % "0.1.0-SNAPSHOT",
```

Comment:
Whoops maybe change it to a Jitpack reference... since currently an attempt to use the plugin as-is ends up in `Error downloading org.virtuslab:sbt-dumpschema-plugin_2.12:0.1.0-SNAPSHOT`, I suppose due to this default

Reply:
I'm gonna repoint `dumpSchemaPlugin` sbt setting to jitpack in Hydra in the meantime...

Reply:
Hmm I've added that to the project as `.settings(dumpSchemaPlugin := "org.virtuslab" %% "sbt-dumpschema-plugin" % "77800371f03baf298bbf638b16a7c65d69b6a9fe")`... but this doesn't get resolved :// I've tagged you on the the relevant Hydra PR